examples/common_vmseries/variables.tf

```diff
@@ -641,8 +703,8 @@ variable "vmseries" {
 
     - `size`              - (`string`, optional, defaults to module default) Azure VM size (type). Consult the *VM-Series
                             Deployment Guide* as only a few selected sizes are supported.
-    - `zone`              - (`string`, optional, defaults to module default) the Availability Zone in which the VM and (if
-                            deployed) public IP addresses will be created.
+    - `zone`              - (`string`, required) the Availability Zone in which the VM and (if deployed) public IP addresses will
```

Comment:
I think `zone` shouldn't be required and vmseries module needs to be fixed to accept zone as optional. In the actual azure vm resource zone is not required.

Reply:
I remember it was a conscious decision to explicitly set `null` as value when you don't want to use availability zones. If we change `zone` to optional, we need to also set the entire `virtual_machine` object as optional, something to think about.

Reply:
We decided to leave it as required on the call.